cms/static/sass/views/_certificates.scss

```diff
@@ -52,7 +52,11 @@
   }
 
   .certificate-info-section{
-    overflow: auto;
+      overflow: auto;
+
+    .course-title-section, .course-number-section{
+      min-width: 50%;
```

Comment:
@frrrances I have made the changes but I had to use `min-width` instead of `width` so that Course Number and Course Title Sections are stacked if one of the two have really long text.